Tools/PerformanceTests/run_alltests_1node.py

```diff
@@ -11,7 +11,14 @@
 import shutil
 import time
 
-from functions_perftest import *
+import pandas as pd
+from functions_perftest import (
```

Comment:
It would probably be better to do something like `import functions_perftest as perftest` and call the routines with that prefix, e.g. `perftest.extract_dataframe`.

Reply:
Thx! Added to #5124, will rebase out here later.